tracpro/polls/tests/test_views.py

```diff
@@ -113,3 +113,55 @@ def test_by_contact(self):
         self.assertEqual(len(responses), 2)
         # newest non-empty first
         self.assertEqual(responses, [self.pollrun2_r1, self.pollrun1_r1])
+
+
+class MultipleResponseCRUDLTest(TracProDataTest):
+
+    def setUp(self):
+        super(MultipleResponseCRUDLTest, self).setUp()
+
+        dates = [
+            datetime.datetime(2014, 1, 1, 6, tzinfo=pytz.UTC),
+            datetime.datetime(2014, 1, 1, 7, tzinfo=pytz.UTC),
+            datetime.datetime(2014, 1, 2, 8, tzinfo=pytz.UTC)
+        ]
+
+        # create pollrun with 3 responses by the same contact (Ann), (2 on the same day, 1 on a different day)
+        self.pollrun = factories.UniversalPollRun(
+            poll=self.poll1, conducted_on=dates[0])
+
+        self.pollrun1_r1 = factories.Response(
+            pollrun=self.pollrun, contact=self.contact1,
+            created_on=dates[0], updated_on=dates[0],
+            status=Response.STATUS_COMPLETE)
+
+        self.pollrun1_r2 = factories.Response(
+            pollrun=self.pollrun, contact=self.contact1,
+            created_on=dates[1], updated_on=dates[1],
+            status=Response.STATUS_COMPLETE)
+
+        self.pollrun1_r3 = factories.Response(
+            pollrun=self.pollrun, contact=self.contact1,
+            created_on=dates[2], updated_on=dates[0],
```

Comment:
looks like a typo here, it's setting updated_on to the first date instead of the third.